portal-impl/src/portal.properties

```diff
@@ -6521,6 +6521,11 @@
     #
     feature.flag.LPS-166918=false
 
+    #
+    # Env: LIFERAY_FEATURE_PERIOD_FLAG_PERIOD__UPPERCASEL__UPPERCASEP__UPPERCASES__MINUS__NUMBER1__NUMBER6__NUMBER6__NUMBER9__NUMBER5__NUMBER4_
+    #
+    feature.flag.LPS-166954=false
```

Comment:
Why are you adding a new FF?

Reply:
The ant format-source-current-branch command added it. Without it we had a ci:test:sf errors.

Reply:
What FF do we use for the Scope Selector ?

Reply:
Yeahh, but we adding the FF here: https://github.com/liferay/liferay-portal/commit/38f3b865c33a0aa1db855e2c030c1df721373612

Reply:
Hmm, now I get it